docs-cn/verify.yaml

```diff
@@ -0,0 +1,87 @@
+version: 1
+
+tasks:
+  - name: check-file-encoding
+    taskType: common
+    shellScript: |
+      git remote add upstream https://github.com/pingcap/docs-cn.git
+      git fetch upstream
+      wget https://raw.githubusercontent.com/CharLotteiu/pingcap-docs-checks/main/check-file-encoding.py
+      python3 check-file-encoding.py $(git diff-tree --name-only --no-commit-id -r upstream/${ghprbTargetBranch}..HEAD -- '*.md' ':(exclude).github/*')
+    buildEnv:
+      image: node:lts
+
+  - name: check-git-conflicts
+    taskType: common
+    shellScript: |
+      git remote add upstream https://github.com/pingcap/docs-cn.git
+      git fetch upstream
+      wget https://raw.githubusercontent.com/CharLotteiu/pingcap-docs-checks/main/check-conflicts.py
+      python3 check-conflicts.py $(git diff-tree --name-only --no-commit-id -r upstream/${ghprbTargetBranch}..HEAD -- '*.md' '*.yml' '*.yaml')
+    buildEnv:
+      image: node:lts
+
+  - name: Lint edited files
+    taskType: common
+    shellScript: |
+      npm install -g markdownlint-cli@0.17.0
+      git remote add upstream https://github.com/pingcap/docs-cn.git
+      git fetch upstream
+      markdownlint $(git diff-tree --name-only --no-commit-id -r upstream/${ghprbTargetBranch}..HEAD -- '*.md' ':(exclude).github/*')
+    buildEnv:
+      image: node:lts
+
+  - name: Check internal links
+    taskType: common
+    shellScript: |
+      wget https://raw.githubusercontent.com/pingcap/docs-cn/ciScripts/scripts/verify-links.sh
+      ./verify-links.sh
+    buildEnv:
+      image: node:lts
+
+  - name: Check internal link anchorss
+    taskType: common
+    shellScript: |
+      wget https://raw.githubusercontent.com/pingcap/docs-cn/ciScripts/scripts/verify-link-anchors.sh
+      ./verify-link-anchors.sh
+    buildEnv:
+      image: node:lts
+
+  - name: "Check control characters"
+    taskType: common
+    shellScript: |
+      git remote add upstream https://github.com/pingcap/docs-cn.git
+      git fetch upstream
+      wget https://raw.githubusercontent.com/CharLotteiu/pingcap-docs-checks/main/check-control-char.py
+      python3 check-control-char.py $(git diff-tree --name-only --no-commit-id -r upstream/${ghprbTargetBranch}..HEAD -- '*.md' ':(exclude).github/*')
```

Comment:
How about change 'ghprbTargetBranch' to  'TARGET_BRANCH', we have changed the env variable name...